### ISSUE TITLE
Argument "3.08_01" isn't numeric in numeric lt (<)

### ISSUE THREAD

**Issue by KES777** [rjbs/Email-Sender#72]
I got next error while trying to send email

    Argument "3.08_01" isn't numeric in numeric lt (<) at Email/Sender/Transport/SMTP.pm line 297

**Comment by KES777** [comment]
near this line this comment:
```
    # For the need to downgrade, see
    #   https://rt.cpan.org/Ticket/Display.html?id=104433
    #
    # The ||0 is there because when we've mocked Net::SMTP, there is no
    # version.  We can't get the ->VERSION call to hit the mock, because we get
    # the mock from ->new.  We don't want to create a new SMTP just to get the
    # version, and we can't rely on $smtp being a Net::SMTP object.
    # -- rjbs, 2015-08-10
    utf8::downgrade($next_hunk) if (Net::SMTP->VERSION || 0) < 3.07;
```

**Comment by ap** [comment]
The fix is to just remove it, per #55

**Comment by rjbs** [comment]
The PR was merged, this issue is defunct.
